README.md

```diff
@@ -76,6 +93,20 @@ and
 [NOTICE](https://github.com/InsightSoftwareConsortium/ITK/blob/master/NOTICE)
 files for details.
 
+Supporting ITK
+--------------
+
+ITK is a fiscally sponsored project of [NumFOCUS](https://numfocus.org/), a non-profit dedicated
+to supporting the open source scientific computing community. If you want to
+support ITK's mission to promote open-source, reproducible scientific image
+analysis software for education and research, please consider making a
+[donation](<!--https://numfocus.org/donate-to-isc-->) to support our efforts.
+
+NumFOCUS is 501(c)(3) non-profit charity in the United States; as such,
+donations to NumFOCUS are tax-deductible as allowed by law. As withany
```

Comment:
withany -> with any

Reply:
Good catch. Thanks Dženan. Done in 5ab5522.